src/NumericalAlgorithms/Spectral/Spectral.hpp

```diff
@@ -172,6 +173,8 @@ T compute_basis_function_value(size_t k, const T& x);
  * \brief Compute the inverse of the weight function \f$w(x)\f$ w.r.t. which
  * the basis functions are orthogonal. See the description of
  * `quadrature_weights(size_t)` for details.
+ * This is arbitrarily set to 1 for FiniteDifference basis, to integrate
```

Comment:
Sorry, why `1`? Or maybe my question is, why is it arbitrary?

Reply:
The only thing that is used is the product of the weights with the inverse weight function. Neither one individually is particularly meaningful for FD. So we set one to the desired value, and the other to 1...

Reply:
I added some comments in the `quadrature_weights(size_t)` function and rebased.

Reply:
Okay, thanks!